examples/Cassie/cassie_encoder.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <memory>
+
+#include <drake/common/eigen_types.h>
+#include <drake/multibody/plant/multibody_plant.h>
+
+#include "systems/framework/output_vector.h"
+
+#include "drake/common/drake_copyable.h"
+#include "drake/systems/framework/leaf_system.h"
+
+namespace dairlib {
+
+#define ENC_RES_LOW 8192
```

Comment:
A bit of a nitpick but I would like to see these named CASSIE_ENC_RES_LOW and _HIGH to avoid future naming conflicts